domain/src/test/java/edu/nextstep/camp/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,55 @@
+package edu.nextstep.camp.calculator
+
+import org.junit.Test
+import com.google.common.truth.Truth.assertThat
+
+class CalculatorTest {
+    @Test
+    fun 문자열_더하기를_했을때_정상적으로_더하기가_되는가() {
+        val calculator = Calculator()
```

Comment:
반복되는 객체생성 작업을 setUp으로 옮겨보면 좋을것 같습니다. 😋

```suggestion
    private lateinit var calculator: Calculator

    @Before
    fun setUp() {
        calculator = Calculator()
    }
```